pkg/controllers/metrics/pods.go

```diff
@@ -0,0 +1,109 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"go.uber.org/multierr"
+	v1 "k8s.io/api/core/v1"
+	crmetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+var (
+	phaseValues = []v1.PodPhase{
+		v1.PodFailed,
+		v1.PodPending,
+		v1.PodRunning,
+		v1.PodSucceeded,
+		v1.PodUnknown,
+	}
+
+	provisionablePodCount = prometheus.NewGauge(
```

Comment:
thoughts on calling this unscheduled? The check is making sure `spec.nodeName =""`, but that doesn't guarantee that it's provisonable.